src/Services.jl

```diff
@@ -1385,11 +1385,19 @@ gamelift(operation, args=[]) =
 gamelift(a...; b...) = gamelift(a..., b)
 
 function glacier(aws::AWSConfig, verb, resource, args=[])
+    version = "2012-06-01"
+    glacier_version_header = ["x-amz-glacier-version"=>version]
+
+    if args isa Pair && args.first == "headers"
+        append!(args[2], glacier_version_header)
```

Comment:
Should probably stick to either `args.first` and `args.second` or `args[1]` and `args[2]`.